corehq/apps/geospatial/static/geospatial/js/geospatial_map.js

```diff
@@ -339,16 +337,7 @@ hqDefine("geospatial/js/geospatial_map", [
                     self.hasFiltersChanged(false);
 
                     // TODO: There is a lot of indexing happening here. This should be replaced with a mapping to make reading it more explicit
```

Comment:
@zandre-eng is this comment still valid? It looks like you've avoided doing what the comment was about ... or have I misunderstood the comment?

Reply:
Good question. I do have another refactor PR which is why I avoided addressing the comment here. Looking through the code however, this function has been simplified a lot, so there really isn't that much indexing happening anymore. I'm thining we can probably remove this comment, unless you have any objections @kaapstorm?

Reply:
I don't have any objections to removing the comment. :)

Reply:
I've made a [separate PR](https://github.com/dimagi/commcare-hq/pull/33996) to remove this comment. Thanks for raising this!